app/main/controllers/template/rtmedia-actions.php

```diff
@@ -790,22 +790,6 @@ function rt_check_addon_status() {
 				}
 			}
 
-			// Listen for activate button to be clicked
-
-			// Also check if information about the addon in already fetched from the store
-			// If it's already fetched, then don't send the request again for the information
-
-			if ( ! empty( $addon_active ) && ! isset( $_POST[ 'edd_' . $addon_id . '_license_activate' ] ) ) {
```

Comment:
Use `filter_input( INPUT_POST, ... )` here, then check for `! empty( $varname )` @umeshnevase